src/material/bottom-sheet/bottom-sheet.md

```diff
@@ -49,7 +49,7 @@ export class HobbitSheet {
 }
 ```
 
-### Configuring bottom sheet content via `entryComponents`
+### Configuring bottom sheet content via `entryComponents` (Since 9.0.0. With Ivy, this is no longer necessary)
```

Comment:
I'm not sure that the title is the proper place to put this. How about right after the title in bold?

Reply:
moved under the title